components/rootless_studio/default/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Create a wrapper to `build` so that any calls to it have a super-stripped
+# `$PATH` and not whatever augmented version is currently in use. This should
```

Comment:
Maybe I'm missing something, but I don't follow how this strips the path. Doing an `exec` replaces the existing shell, but doesn't destroy the environment. Maybe you want [`exec -c`](https://www.gnu.org/software/bash/manual/html_node/Bourne-Shell-Builtins.html#index-exec) or `env -i`?

Reply:
I copied this from https://github.com/habitat-sh/habitat/blob/master/components/studio/libexec/hab-studio-type-default.sh#L113-L122

Reply:
Ah, ok. Well, I guess I don't understand how that's working either, but if it's working for you, I can't really argue with that.

@fnichol: I think you wrote the original code. Is there something I'm missing?

Reply:
I'll look here shortly…